commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -378,6 +379,18 @@ def save_inventory(environment, inventory):
           file=sys.stderr)
 
 
+def save_vault_yml(environment):
+    def generate_uuid():
+        return str(uuid.uuid4())
+    j2.globals.update(generate_uuid=generate_uuid)
+    template = j2.get_template('private.yml.j2')
+    vault_file_contents = template.render(my_pw="elephant")
```

Comment:
what's `my_pw` for? I don't see it referenced in the template.

Reply:
Oops, I was using that for debugging, removed here: https://github.com/dimagi/commcare-cloud/pull/2040/commits/89f1837f78a6f308372f720bc3019df22cb82667